src/components/VolunteersContactMethods/ContactMethod.js

```diff
@@ -0,0 +1,23 @@
+/* jshint ignore: start */
+import React from 'react';
+import './ContactMethod.css';
+
+const ContactMethod = ({ data }) => {
+  const method = data;
```

Comment:
is there a reason to call data method here?
and I strongly advice you to destructure data